infrastructure/README.md

```diff
@@ -1,5 +1,5 @@
 
-# Project Jackson Infrastructure
+# Project Infrastructure
 
 This repository includes the ARM templates for Project Jackson.
```

Comment:
Hi @SharePointing - If we're going to take this change, there are some other places throughout the docs that should have this removed as well - like this one

Reply:
I guess we have to decide - is it Project Jackson or "Containers Rest Cosmos Java"

I was just simplifying to "Project in the generic sense" 

Happy to leave it be if we want it to be consistent, I'm just starting to point people here as reference sometimes

Reply:
@code4clouds can you comment on the overall project goal? If we want to rebrand we should book the overall work to do so, rather than asking @SharePointing to do it here. 😃